src/shared/util/verification.ts

```diff
@@ -21,3 +21,5 @@ export enum VfnErrors {
   TransactionNotFound = 'TRANSACTION_NOT_FOUND',
   InvalidMobileNumber = 'INVALID_MOBILE_NUMBER',
 }
+
+export const SMS_VERIFICATION_LIMIT = 10000
```

Comment:
should we make this an env var with a default and instantiate in config?

We can still send the variable to the frontend just like how we send other variables currently

Reply:
sure, changed it and updated default to 10,000!